cmake/features.cmake

```diff
@@ -45,15 +45,15 @@ ie_option(ENABLE_ERROR_HIGHLIGHT "Highlight errors and warnings during compile t
 
 # Try to find python3
 find_package(PythonLibs 3 QUIET)
-ie_dependent_option (ENABLE_PYTHON "enables ie python bridge build" OFF "PYTHONLIBS_FOUND" OFF)
+ie_dependent_option (ENABLE_PYTHON "enables ie python bridge build" ON "PYTHONLIBS_FOUND" OFF)
 
 find_package(PythonInterp 3 QUIET)
 ie_dependent_option (ENABLE_DOCS "Build docs using Doxygen" OFF "PYTHONINTERP_FOUND" OFF)
 
 # this option should not be a part of InferenceEngineDeveloperPackage
 # since wheels can be built only together with main OV build
-cmake_dependent_option (ENABLE_WHEEL "Build wheel packages for PyPi" OFF
-    "PYTHONINTERP_FOUND;CMAKE_SOURCE_DIR STREQUAL OpenVINO_SOURCE_DIR" OFF)
+cmake_dependent_option (ENABLE_WHEEL "Build wheel packages for PyPi" ON
```

Comment:
should we `ENABLE_WHEEL` iff `ie_check_pip_package` has passed?

Reply:
added ov_check_pip_packages function to check requirements.txt files
cmake/developer_package/IEDevScriptsConfig.cmake